src/main/java/org/skife/config/Default.java

```diff
@@ -10,4 +10,5 @@
 public @interface Default
 {
     String value();
+    String[] for_values() default "*";
```

Comment:
Make this "" so that it will never match

Reply:
Or make this null.

Reply:
Please also verify with a test what for_values returns when the default is used (eg. as in `@Default("foo")`)